tools/src/main/java/com/nedap/archie/archetypevalidator/validations/SpecializedDefinitionValidation.java

```diff
@@ -109,6 +111,27 @@ private boolean hasAssertions(List<Assertion> assertions) {
     private void validateConformsTo(CObject cObject, CObject parentCObject) {
 
         ConformanceCheckResult conformanceCheckResult = cObject.cConformsTo(parentCObject, combinedModels::rmTypesConformant);
+        if (!conformanceCheckResult.doesConform() && conformanceCheckResult.getErrorType().equals(ErrorType.VSONCO)) {
+            // Edge case: if sum of all occurrences of all redefined object nodes conforms to occurrences of parent object node
+            // this is valid and should not result in the error
+            MultiplicityInterval parentOccurrences = parentCObject.effectiveOccurrences(combinedModels::referenceModelPropMultiplicity);
+            List<CObject> allRedefinedCObjects = cObject.getParent().getChildren().stream().filter(
```

Comment:
looks good. However, I'm not sure the following situation is catched here:

parent-archetype:
```
ELEMENT[id5] occurrences matches {2..8}
```

first specialization (differential/source format):
```
ELEMENT[id5.1] occurrences matches {2}
```

second specialization (differential/source format):

```
ELEMENT[id5.0.1] occurrences matches {0..1}
```